### PR TITLE
Remove obsolete plugins

### PR DESCRIPTION
**Pull request by javanlacerda**
- Remove absolete plugins

### CODE REVIEW

File: controller/plugins/actuator/builder.py [hunk 1]
```diff
@@ -13,131 +13,18 @@
...
-            return ServiceActuator(actuator_port, instance_locator)
-
-        elif name == "external_api":
-            actuator_metric = api.actuator_metric
-            actuator = ExternalApi(parameters['app_id'],
-                                   actuator_metric, api.k8s_manifest)
-
             return actuator
 
         elif name == "nop":
```
Comment (joseims) [inline controller/plugins/actuator/builder.py]: Is this still necessary?

File: controller/plugins/metric_source/builder.py [hunk 2]
```diff
@@ -15,24 +15,15 @@
...
-    OpenstackGenericMetricSource
-)
 
 
 class MetricSourceBuilder:
     def get_metric_source(self, name, parameters):
         if name == "monasca":
             return MonascaMetricSource(parameters)
-
         elif name == "nop":
```
Comment (joseims) [inline controller/plugins/metric_source/builder.py]: Ditto

File: controller/plugins/actuator/builder.py (hunk 1 above)
Comment (javanlacerda) [reply controller/plugins/actuator/builder.py]: Yes. The unit tests use this actuator.

File: controller/plugins/metric_source/builder.py (hunk 2 above)
Comment (javanlacerda) [reply controller/plugins/metric_source/builder.py]: Ditto

**Pull request merged**